resources/variants/zyyx/zyyx_pro_carbon0.6.inst.cfg

```diff
@@ -0,0 +1,18 @@
+[general]
+name = Carbon0.6
+version = 4
+definition = zyyx_pro
+
+[metadata]
+setting_version = 3
+type = variant
+hardware_type = nozzle
+
+[values]
+acceleration_enabled = True # No effect on print, only for better print time prediction
+acceleration_print = 1000
+line_width = 0.6
```

Comment:
This is not necessary. `line_width` is by default equal to `machine_nozzle_size`, which you already properly define. Same for other variants.